### DISCUSSION TITLE
mongo副本级连接失败

### DISCUSSION THREAD

**Discussion by ChengJiuSu** [hhyo/Archery#1725] [Q&A]
各位技术大佬，希望能得到帮助。
archery 连接mongodb+srv 地址提示无法解析：
无法连接实例，
报错如下
archery-test.rmgc9.mongodb.net:27017: [Errno-2] Name or
service not known, Timeout: 30s, Topology Description:
<TopologyDescription id: 62e8e54cb5441aa3d9a6c99f,
topology_type: Single, servers: [«ServerDescription ('archery-
test.rmgc9.mongodb.net', 27017) server_type: Unknown, rtt:
None, error=AutoReconnect ('archery-
test.rmgc9.mongodb.net:27017: [Errno-2] Name or service not
known'>]>

archery连接mongodb+srv解析的地址提示连接失败。当前机器是可以通过  mongo "mongodb://" 或mongo "mongodb+srv://" 方式连接；
报错如下：
无法连接实例，
connection closed, Timeout: 30s, Topology Description:
<TopologyDescription id: 62e8e265b5441aa3d9a6c99c,
topology_type: Single, servers: [<ServerDescription ('archery-test-
shard-00-02.rmgc9.mongodb.net', 27017) server_type: Unknown,
rtt: None, error=AutoReconnect ('connection closed') >]

**Comment by czxin788** [comment]
我也遇到类似问题了，
写一行语句就行，写两行就报错
![image](https://user-images.githubusercontent.com/13861904/187458547-e83bad66-4862-4f67-8ca2-cff8e3ffd8e1.png)
![image](https://user-images.githubusercontent.com/13861904/187459083-e41c888d-4409-49b4-95c7-92c82c5c2780.png)

**Comment by czxin788** [comment]
还有个类似的issue，
https://github.com/hhyo/Archery/issues/1138
